app/core/services/PageBuilder.scala

```diff
@@ -207,20 +208,30 @@ class PageBuilder @Inject() (val placeholders: Placeholders) extends ProcessPopu
     }
 
   @tailrec
-  private def checkExclusiveSequencePages(pages: Seq[Page], errors: List[GuidanceError]): List[GuidanceError] = {
+  private def checkExclusiveSequenceTypeError(pages: Seq[Page], errors: List[GuidanceError]): List[GuidanceError] =
+    pages match {
+      case Nil => errors
+      case p :: xs if p.stanzas.collectFirst{case _: ExclusiveSequence => Unit}.fold(false)(_ => true) &&
```

Comment:
Is it better to use () rather than Unit? I tried a similar construct in REPL and it complained about the use of Unit in source code.

Reply:
updated